src/main/java/hudson/plugins/logparser/LogParserAction.java

```diff
@@ -59,7 +64,7 @@ public static String getUrlNameStat() {
         return urlName;
     }
 
-    public AbstractBuild<?, ?> getOwner() {
+    public Run<?, ?> getOwner() {

```

Comment:
I'm pretty sure I've used these actions in my scripts at the previous work... And yes, the plugin used to work well :)

BTW, a release with a new major version could be an option.